### PR TITLE
Fix description icon sizes

### PR DESCRIPTION
**Pull request by ma12-co**
Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

**Comment by PVince81** [comment]
seems the icons were not vertically centered on the buttons before:
image

and now when smaller it makes it slightly more obvious:
image

**Comment by ma12-co** [comment]
Fixed!

File: css/icons.scss [hunk 1]
```diff
@@ -113,6 +113,11 @@
 			filter: initial;
 		}
 	}
+
+	// Patch material design icons' span wrong height
+	.material-design-icon {
+		display: contents;
```
Comment (PVince81) [inline css/icons.scss:119]: risky change, not sure if this is intended (in call sidebar):

before:

image

after:

image

Comment (PVince81) [reply css/icons.scss:119]: oh, apparently it also reduces the emoji icon's side in the regular messages view, so it's not about narrow panels

**Comment by PVince81** [comment]
the regression is now gone

but now the buttons are too close to that blue bottom bar:
image

what is that bar anyway ? maybe the border of the field, we could try and hide it

**Comment by PVince81** [comment]
image

File: css/icons.scss (hunk 1 above)
Comment (ma12-co) [reply css/icons.scss:119]: This I fixed

**Comment by ma12-co** [comment]
Margins fixed, ready to review again!

**Comment by PVince81** [comment]
the blue bar is still there, but as it was there before I guess it's intentional.
the distance from the bar looks fine now:
image

image

**Review by PVince81 — APPROVED** [review]
:+1:

**Pull request merged**